src/Resources/views/Block/block_admin_preview.html.twig

```diff
@@ -0,0 +1,85 @@
+{#
+
+This file is part of the Sonata package.
+
+(c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+
+For the full copyright and license information, please view the LICENSE
+file that was distributed with this source code.
+
+#}
+
+{% extends sonata_block.templates.block_base %}
+
+{% set translation_domain = settings.translation_domain ?? admin.translationDomain %}
+
+{% block block %}
+    {% set inlineAnchor = settings.text|replace({'.':'_'}) %}
+    <div class="box box-primary" id="{{ inlineAnchor }}">
+        <div class="box-header with-border">
+            {% set icon = settings.icon|default('') %}
+            {% if icon %}
+                <i class="fa {{ icon|raw }}"></i>
+            {% endif %}
+            <h3 class="box-title">
+                <a href="#{{ inlineAnchor }}">{{ settings.text|trans({}, translation_domain) }}</a>
+            </h3>
+        </div>
+        <div class="box-body {% if admin.datagrid.results|length > 0 %}table-responsive no-padding{% endif %}">
+            {{ sonata_block_render_event('sonata.admin.list.table.top', { 'admin': admin }) }}
+
+            {% block list_header %}{% endblock %}
+
+            {% if admin.datagrid.results|length > 0 %}
+                <table class="table table-bordered table-striped table-hover sonata-ba-list">
+                    {% block table_header %}
+                        <thead>
+                            <tr class="sonata-ba-list-field-header">
+                                {% for field_description in admin.list.elements %}
+                                    {% if field_description.getOption('code') == '_select' %}
```

Comment:
I think `code` option was deprecated and there was a constant for the `select` part cc @VincentLanglet